rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -70,11 +73,11 @@
     primitive_types = [bool, int, long, float]
     python2 = True
 
-list_types = [list, tuple]
+list_types = [list, tuple, np.ndarray, array.array]
```

Comment:
Oh nice, you have also fixed arrays.

Reply:
Can't take credit for this one, that was done by the upstream PR that's been stuck "in review"